app/Models/Season.php

```diff
@@ -45,6 +48,28 @@ public static function latestOrId($id)
         return $season;
     }
 
+    public function divisions(): HasMany
+    {
+        return $this->hasMany(SeasonDivision::class);
+    }
+
+    public function divisionsWithMaxRanks(?int $userCount = null): array
+    {
+        $divisions = $this->divisions()->orderBy('threshold')->get();
+        $userCount ??= $this->userScores()->forRanking()->count();
+
+        $divisionsWithMaxRanks = [];
+
+        foreach ($divisions as $division) {
+            $divisionsWithMaxRanks[] = [
+                'division' => $division,
+                'max_rank' => get_int($division->threshold * $userCount),
```

Comment:
it works but it's not quite what get_int is for... ಠ_ಠ (it's more for validating unknown variable/string, not a known float)